cli/src/main/java/org/jboss/gm/cli/Main.java

```diff
@@ -198,6 +199,8 @@ private void executeGradle() throws ManipulationException {
             Set<String> jvmArgs = jvmPropertyParams.entrySet().stream()
                     .map(entry -> "-D" + entry.getKey() + '=' + entry.getValue())
                     .collect(Collectors.toCollection(LinkedHashSet::new));
+            jvmArgs.addAll(ManagementFactory.getRuntimeMXBean().getInputArguments().stream()
+                    .filter(s -> s.startsWith("-Xdebug") || s.startsWith("-Xrunjdwp")).collect(Collectors.toSet()));
```

Comment:
Can you explain the reason for this line?

Reply:
If I'm trying to debug a GME run, if I do it through the CLI, it doesn't debug the spawned Gradle process - which is what I really wanted in the first place. So this propagates those CLI parameters so that the Gradle alignment run may be debugged.